src/pages/Login/ResetPassword.styles.scss

```diff
@@ -0,0 +1,65 @@
+$dark: #6831DE;
+$light: #E03BFB;
```

Comment:
The same thing I mentioned above regarding variables.

Reply:
Good point! It's done now!